### DISCUSSION TITLE
Query regarding Taiko Gauge Framework for Database validation

### DISCUSSION THREAD

**Discussion by ankitme612** [getgauge/taiko#2109] [General]
Hi Team,

I have recently started using taiko +Gauge framework for UI Automation. I have this query, can we implement database validations as well in this framework? For example you are creating something from frontend and validate from the database it has created or not

**Comment by zabil** [comment]
Gauge and Taiko runs on Node js. So you can use any node js library to connect to the database and read values in a Gauge step implementation.
